etc/testing/test_worker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euxo pipefail
+
+export TIMEOUT="${1}"
```

Comment:
Would be nice to error with a helpful message if the timeout is omitted (or just have a default value).

Reply:
Default added by #5374